corehq/apps/cleanup/management/commands/fix_forms_with_missing_xmlns.py

```diff
@@ -0,0 +1,93 @@
+from corehq.apps.app_manager.models import Application
+from corehq.apps.app_manager.xform import XForm, parse_xml
+from couchforms.models import XFormInstance
+from dimagi.utils.couch.database import iter_docs
+from django.core.management.base import BaseCommand
+from optparse import make_option
+
+
+class Command(BaseCommand):
+    help = 'fix all of the things'
+
+    option_list = BaseCommand.option_list + (
+        make_option(
+            '--dry-run',
+            action='store_true',
+            dest='dry_run',
+            default=False,
+            help="Don't do the actual modifications, just print what would be affected"
+        ),
+    )
+
+    def handle(self, *args, **options):
+
+        dry_run = options.get("dry_run", True)
+        new_xmlnss = {}
+
+        for xform_instance in get_submissions_without_xmlns():
+            if xform_instance.app_id in new_xmlnss:
+                set_xmlns_on_submission(xform_instance, new_xmlnss[xform_instance.app_id], dry_run)
+            else:
+                app = Application.get(xform_instance.app_id)
+                forms_without_xmlns = get_forms_without_xmlns(app)
+                if len(forms_without_xmlns) == 1:
+                    new_xmlns = generate_random_xmlns()
```

Comment:
The only property that xmlns needs to have is that it is unique from other xmlns's in the app.